x-pack/plugins/observability_solution/observability_ai_assistant/server/service/client/adapters/process_bedrock_stream.ts

```diff
@@ -97,7 +101,7 @@ export function processBedrockStream({
           completion = before.trimEnd();
         } else if (isEndOfFunctionCall) {
           completion = '';
-          functionCallsBuffer += chunkBody.completion + chunkBody.stop;
+          functionCallsBuffer += prepareBedrockOutput(chunkBody) + chunkBody?.delta?.stop_sequence;
```

Comment:
Can we reuse here the variable from L86? 
`let completion = prepareBedrockOutput(chunkBody);`

Reply:
but look, on L103 completion is set to `''`. This is o11y code, will defer to Dario for these cleanups who is reviewing later today